doc/news/changes/minor/20191009NiklasFehn

```diff
@@ -0,0 +1,4 @@
+New: Write new function in GridTools to compute the maximum aspect ratio 
```

Comment:
```suggestion
New: The function GridTools::calculate_aspect_ratio_of_cells() computes the maximum aspect ratio 
```

Reply:
I can change this. I am somehow hesitating because the change of name as requested below will introduce inconsistencies if not remembering all occurrences. Should I change nevertheless?

Reply:
It is useful to have the name of the new function in the changelog and "Write new function ..." sounds a bit weird. Of course they will need updating before we merge the PR to make sure they are consistent. You are welcome to wait to address this comment until everything else is figured out.

Reply:
changed in https://github.com/dealii/dealii/pull/8894/commits/957deb80a0ed01b949663db78708c52e2f643f33